Extension/src/LanguageServer/settingsTracker.ts

```diff
@@ -38,12 +39,14 @@ export class SettingsTracker {
     private collectSettings(filter: FilterFunction): { [key: string]: string } {
         const settingsResourceScope: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration("C_Cpp", this.resource);
         const settingsNonScoped: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration("C_Cpp");
-        const selectCorrectlyScopedSettings = (rawSetting: any): vscode.WorkspaceConfiguration =>
+        const caseSensitiveFileSupportLastState: PersistentWorkspaceState<string> = new PersistentWorkspaceState<string>("CPP.isFilePathHandlingCaseSensitivePersistent", "default");
```

Comment:
>isFilePathHandlingCaseSensitivePersistent [](http://example.com/codeflow?start=126&length=41)

The name of the state variable should be updated to reflect the new setting name.

'CPP.currentCaseSensitiveFileSupport' or something like that.